styles/Booklist.module.css

```diff
@@ -0,0 +1,57 @@
+.card {
+  margin: 1rem;

```

Comment:
Is this intentional? Search up rem, em, and px. I think it's better to use px to be "pixel-perfect" to the design. Em will scale w/ fonts and the parent container. Just keep that in mind.